README.md

```diff
@@ -1 +1,31 @@
-# Flashcard-Maker
\ No newline at end of file
+# Flashcard-Maker
+
+## Deck Creation DSL
+### Example
+```
+Create Deck French with Tags language, hard, midterm using Style mystyle:  
+(1) Front:Back
+(2) Front:Back
+(3) Front:Back
+
+Create Style mystyle:
+Color = Red
+Direction = Horizontal
+Align = Center
+```
+
+### EBNF
+```
+CREATE_DECK ::= “Create Deck” NAME (“with Tags“ TAGS | “using Style” STYLE)*”:” DECK
+DECK ::= CARD+
+CARD ::= “(”NUMBER”) ” STRING “:” STRING
+TAGS ::= STRING”,”? TAGS
+
+CREATE_STYLE ::= “Create Style” NAME “:” STYLE
+STYLE ::= ATTRIBUTE “=” STRING
+ATTRIBUTE ::= “Direction” | “Font” | “Align” | “Color” | “Colour”
```

Comment:
We should make the right hand side of STYLE more specific (like what we have in the Google doc), not just STRING
If user specifies a string which we cannot parse, it would cause an error right?

Reply:
I feel like we can leave that error checking to the evaluate step since it would probably be easier to implement there. Otherwise, it would just make our EBNF way more complex (like having to list all the different possible colors)

Reply:
I think for color we can use STRING but other styling attributes which are limited to 2-3 options can be specified in EBNF. But I guess there's not much difference whether we handle the errors in parse or evaluation so probably it's fine